FakeItEasy.Deploy/Program.cs

```diff
@@ -8,10 +8,9 @@
     using FakeItEasy.Tools;
     using Octokit;
     using static FakeItEasy.Tools.ReleaseHelpers;
-    using static FakeItEasy.Tools.ToolHelpers;
     using static SimpleExec.Command;
 
-    public class Program
+    internal static class Program
```

Comment:
In case you were wondering: I made it internal, because when it was public, I got a warning for not validating that `args` wasn't null in `Main`....